tensorflow_datasets/image/voc.py

```diff
@@ -193,3 +221,78 @@ def _generate_example(self, data_path, image_id):
         "labels": labels,
         "labels_no_difficult": labels_no_difficult,
     }
+
+class Voc2012(tfds.core.GeneratorBasedBuilder):
+  """Pascal VOC 2012."""
+
+  VERSION = tfds.core.Version("1.0.0",
+                              experiments={tfds.core.Experiment.S3: False})
+  SUPPORTED_VERSIONS = [
+      tfds.core.Version("3.0.0"),
+      tfds.core.Version("2.0.0"),
+  ]
+
```

Comment:
For new datasets use only a single new version. Replace with only `VERSION = tfds.core.Version("1.0.0")`